marginaleffects/datagrid.py

```diff
@@ -11,6 +11,7 @@ def datagrid(
     grid_type="mean_or_mode",
     FUN_numeric=lambda x: x.mean(),
     FUN_other=lambda x: x.mode()[0],  # mode can return multiple values
+    FUN_categorical=lambda x: x.unique(),
```

Comment:
by default, with `mean_or_mode`, categorical variables should be summarized with the mode, not as unique values. Unique values should be use when the user requests them and for `newdata="balanced"`